### PR TITLE
Added OTE real-life training tests

### PR DESCRIPTION
**Pull request by pfinashx**
Added OTE real-life training tests for Custom_Semantic_Segmentation_Lite-HRNet-18_OCR

### CODE REVIEW

File: mmseg/apis/ote/apis/segmentation/nncf_task.py [hunk 1]
```diff
@@ -65,6 +66,7 @@ def __init__(self, task_environment: TaskEnvironment):
         self._val_dataloader = None
         self._compression_ctrl = None
         self._nncf_preset = "nncf_quantization"
+        self._optimization_type = ModelOptimizationType.NNCF
```
Comment (alexsu52) [inline mmseg/apis/ote/apis/segmentation/nncf_task.py]: Please, add this line after super().__init__(task_environment)

Comment (pfinashx) [reply mmseg/apis/ote/apis/segmentation/nncf_task.py]: Done, thanks!

**Review by alexsu52 — APPROVED** [review]

**Review by evgeny-izutov — APPROVED** [review]

**Review by evgeny-izutov — APPROVED** [review]

File: mmseg/apis/ote/apis/segmentation/nncf_task.py [hunk 2]
```diff
@@ -212,6 +214,8 @@ def optimize(
         output_model.optimization_type = self._optimization_type
         output_model.optimization_methods = self._optimization_methods
         output_model.precision = self._precision
+        print(f"!!!self._optimization_type = {self._optimization_type}")
+        print(f"!!!output_model.optimization_type = {output_model.optimization_type}")
```
Comment (LeonidBeynenson) [inline mmseg/apis/ote/apis/segmentation/nncf_task.py]: Please, do not forget to remove that!

Comment (pfinashx) [reply mmseg/apis/ote/apis/segmentation/nncf_task.py]: Already done.

**Review by LeonidBeynenson — APPROVED** [review]

**Pull request merged**